app/data/15.3.yml.erb

```diff
@@ -29,6 +30,7 @@
       - name: <%= _("Torrent File") %>
         url: /distribution/leap/<%= @version %>/iso/openSUSE-Leap-<%= @version %>-NET-x86_64-Current.iso.torrent
   - name: aarch64
+    desc: UEFI Arm 64-bit servers, desktops, laptops and boards
```

Comment:
ARM, not Arm

Reply:
No, this is "Arm" ;)

Reply:
OK, so looking this up, apparently there are 3 schools:
* arm (logo stylistic)
* ARM (abbreviation)
* Arm (after Arm Holdings)

So I say it doesn't matter because every way is wrong :P

Reply:
I know in my experience, that ARM has changed the name in the logo to arm. The employee wants to use a mixture. Should we use the name as it is written in the real company name? https://www.arm.com/company/policies/trademarks

Reply:
We should use `Arm`